### PR TITLE
Removed some redundant metrics and added more metrics for telemeter.

### PR DESCRIPTION
**Pull request by anmolsachan**
Signed-off-by: Anmol Sachan <anmol13694@gmail.com>

### CODE REVIEW

**Review by shtripat — COMMENTED** [review]
@pcuzner please check as @anmolsachan mentioned to you.
If these are redundant we can remove these.

File: extras/manifests/prometheus-ceph-rules.yaml [hunk 1]
```diff
@@ -18,54 +18,21 @@ spec:
...
+        count(kube_persistentvolume_info)
+      record: job:kube_pv:count
+    - expr: |
+        sum(ceph_pool_rd{job="rook-ceph-mgr"}+ ceph_pool_wr{job="rook-ceph-mgr"})
+      record: job:ceph_pools_iops:total
     - expr: |
-        count(ceph_rgw_metadata{job="rook-ceph-mgr"})
-      record: job:ceph_rgw_metadata:count
+        sum(ceph_pool_rd_bytes{job="rook-ceph-mgr"}+ ceph_pool_wr_bytes{job="rook-ceph-mgr"})
+      record: job:ceph_pools_iops_bytes:total
```
Comment (shtripat) [inline extras/manifests/prometheus-ceph-rules.yaml:32]: Do we call this as iops_bytes or `throughput`

Comment (anmolsachan) [reply extras/manifests/prometheus-ceph-rules.yaml:32]: Throughput is change per second and this is instantaneous value. Thats why I kept it like this. Should we change ?

Comment (shtripat) [reply extras/manifests/prometheus-ceph-rules.yaml:32]: Looks fine then.

File: rules/rules.libsonnet [hunk 2]
```diff
@@ -21,128 +21,128 @@
...
+              count(count(ceph_mon_metadata{%(cephExporterSelector)s} or ceph_osd_metadata{%(cephExporterSelector)s} or ceph_rgw_metadata{%(cephExporterSelector)s} or ceph_mds_metadata{%(cephExporterSelector)s} or ceph_mgr_metadata{%(cephExporterSelector)s}) by(ceph_version))
             ||| % $._config,
           },
         ],
       },
     ],
   },
 }
+          // {
+          //   record: 'job:ceph_osd_in:count',
```
Comment (shtripat) [inline rules/rules.libsonnet]: You want to keep these lines commented?
Remove this.

Comment (anmolsachan) [reply rules/rules.libsonnet]: I would say, lets have these commented and be here. We might need some of these later

Comment (shtripat) [reply rules/rules.libsonnet]: Ok. Still looks bad. Anyway.

**Review by shtripat — APPROVED** [review]
LGTM

**Pull request merged**